lib/src/main/kotlin/com/lemonappdev/konsist/core/declaration/KoProperty.kt

```diff
@@ -31,25 +32,21 @@ class KoProperty private constructor(private val ktProperty: KtProperty) : KoDec
         type?.let { KoType.getInstance(it) }
     }
 
-    fun hasLateinitModifier() = ktProperty.modifierList?.hasModifier(KtTokens.LATEINIT_KEYWORD) ?: false
+    fun hasLateinitModifier() = hasModifiers(KoModifier.LATEINIT)
 
-    fun hasOverrideModifier() = ktProperty.modifierList?.hasModifier(KtTokens.OVERRIDE_KEYWORD) ?: false
+    fun hasOverrideModifier() = hasModifiers(KoModifier.OVERRIDE)
 
-    fun hasAbstractModifier() = ktProperty.modifierList?.hasModifier(KtTokens.ABSTRACT_KEYWORD) ?: false
+    fun hasAbstractModifier() = hasModifiers(KoModifier.ABSTRACT)
 
-    fun hasOpenModifier() = ktProperty.modifierList?.hasModifier(KtTokens.OPEN_KEYWORD) ?: false
+    fun hasOpenModifier() = hasModifiers(KoModifier.OPEN)
 
-    fun hasFinalModifier() = ktProperty.modifierList?.hasModifier(KtTokens.FINAL_KEYWORD) ?: false
+    fun hasFinalModifier() = hasModifiers(KoModifier.FINAL)
 
-    fun hasActualModifier() = ktProperty.modifierList?.hasModifier(KtTokens.ACTUAL_KEYWORD) ?: false
+    fun hasActualModifier() = ktProperty.hasActualModifier()
```

Comment:
Let's use hasModifiers(KoModifier.X) for every modifier property because "kt methods" may have additional behavior